src/Sylius/Behat/Context/Api/Shop/OrderContext.php

```diff
@@ -122,6 +128,94 @@ public function iShouldSeeAsShippingAddress(
         Assert::same($address['countryCode'], $country->getCode());
     }
 
+    /**
+     * @Then I should see :amount items in the list
+     */
+    public function iShouldSeeItemsInTheList(int $amount): void
+    {
+        Assert::same(count($this->responseChecker->getValue($this->client->getLastResponse(), 'items')), $amount);
+    }
+
+    /**
+     * @Then the product named :productName should be in the items list
+     */
+    public function theProductShouldBeInTheItemsList(string $productName): void
+    {
+        $items = $this->responseChecker->getValue($this->client->getLastResponse(), 'items');
+
+        foreach ($items as $item) {
+            if ($item['productName'] === $productName) {
+                Assert::true(true);
+
+                return;
+            }
+        }
+
+        Assert::true(false);
+    }
+
+    /**
+     * @Then /^the (shipment) status should be "([^"]+)"$/
+     * @Then /^I should see its (payment) status as "([^"]+)"$/
+     */
+    public function theShipmentStatusShouldBe(
+        string $elementType,
+        string $elementStatus,
+        int $numberOfelement = 0
+    ): void {
+        $shipments = $this->responseChecker->getValue($this->client->getLastResponse(), $elementType . 's');
```

Comment:
```suggestion
        $resources = $this->responseChecker->getValue($this->client->getLastResponse(), $elementType . 's');
```
as it could be shipment or payment